.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
+  "parserOptions": {
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true
+    }
+  },
+  "extends": ["prettier"],
+  "env": {
+    "es6": true,
+    "node": true
+  }
+}
```

Comment:
This file has been created to support prettier formatting rules when processing JS files with eslint (see scripts directory).